workers/unity/Packages/com.improbable.gdk.deploymentlauncher/.DeploymentLauncher/Ipc.cs

```diff
@@ -53,11 +53,24 @@ private struct InternalDeployment
         {
             public string Id;
             public string Name;
+            public string StartTime;
+            public string Region;
+            public List<string> Tags;
+            public Dictionary<string, int> Workers;
 
             public InternalDeployment(Deployment deployment)
             {
                 Id = deployment.Id;
                 Name = deployment.Name;
+
+                var epoch = new DateTime(1970, 1, 1, 0, 0, 0, DateTimeKind.Utc);
```

Comment:
Why not `DateTime.MinValue`?

Reply:
Because I didn't know that was a thing 😅

Reply:
Actually `DateTime.MinValue` is: 

> equivalent to 00:00:00.0000000 UTC, January 1, 0001, in the Gregorian calendar

But! There is also a simpler way to do this